lib/wasi/src/runners/mod.rs

```diff
@@ -107,11 +107,11 @@ impl Bindings for WitBindings {
         container: &WapmContainer,
         value: &serde_cbor::Value,
     ) -> Result<Self, String> {
-        let value: webc::WitBindingsExtended =
+        let value: webc::BindingsExtended =
```

Comment:
This seems wrong. Am I missing something?

Reply:
cc @Michael-F-Bryan

Reply:
That's the latest release.

https://docs.rs/webc/latest/webc/enum.BindingsExtended.html